### PR TITLE
Feature/add iis server

### PR DESCRIPTION
**Pull request by ashim-mahara**
- Bootstrap key is added at the start when initializing the ansible instance
 - No longer need the ad collections since it comes with ansible via pip
 - Reintroduced the common_windows_vars
 - Updated Tasks for the IIS Server
 - Lowered the delay time for the IIS Server role when checking for connection
 - Remove the default website from the IIS Servers
 - Added support to copy the build file using SCP and SSH within terraform -> This is a biggie
 - For the SCP, the configs should be in place for the Jump. Need to add this later.

### CODE REVIEW

File: terraform/gif-website/build/robots.txt
Comment (gidoBOSSftw5731) [inline terraform/gif-website/build/robots.txt:1]: Consider leveraging this in the future to do something interesting

File: terraform/socInstances.tf [hunk 1]
```diff
@@ -51,6 +64,37 @@ resource "openstack_compute_instance_v2" "ansible" {
...
+  depends_on = [
+    data.archive_file.build_zip
+  ]
+}
+
+resource "null_resource" "copy_build_zip" {
+  depends_on = [openstack_compute_instance_v2.ansible, data.archive_file.build_zip]
+
+  provisioner "local-exec" {
+    command = <<-EOT
```
Comment (gidoBOSSftw5731) [inline terraform/socInstances.tf:77]: Letting this stay but if it gets any bigger please move it to another file

**Review by gidoBOSSftw5731 — APPROVED** [review]
A few semantics errors but overall looks good

File: terraform/socInstances.tf (hunk 1 above)
Comment (ashim-mahara) [reply terraform/socInstances.tf:77]: okay

**Pull request merged**